src/Helpers/IPFSClient.test.ts

```diff
@@ -0,0 +1,28 @@
+import { IPFS } from './IPFSClient';
+
+const port = 5001
+const ipfsUrl = `http://localhost:${port}`;
```

Comment:
we'll need a way to override 'port' and 'ipfsUrl' with a env var - currently the test won't run from inside the node container, only outside.

Reply:
This isn't a real test file, and the values are just hard coded on purpose for the moment so I could expirement with the differences.

do we want integration tests for these types of files? If so more than happy to create the integration tests.

If so I will need an ipfs executable available to start from my test file or the ipfs container running

Reply:
Here is the original pr that uses the api directly https://github.com/poetapp/node/pull/215